README.md

```diff
@@ -227,7 +227,7 @@ The `onSuccess` and `onError` callbacks are optional.
         },
         // This error callback is optional and gets called when an error occurs when the app is sending the order
         onError = { error ->
-            Log.e(TAG, error.message())
+            Log.e(TAG, error.message)
```

Comment:
Sorry Android noob here, shouldn't it be like below:

```
error.getMessage()
```

Reply:
There are some differences between Kotlin and Java. I'm not 100% sure why it works like that. But basically it's working like the following:

The code in Kotlin to declare the VirtusizeError class is 
`data class VirtusizeError(val type: VirtusizeErrorType? = null, val code: Int? = null, val message: String)`
We access its properties directly when we are just using this class in Kotlin.
So it should be `error.message` in Kotlin

But if we use this Kotlin class in Java, the compiler generates the getters for those properties and the properties will be private instead. 
So in Java, it will be like `error.getMessage()`